.idea/sonarlint/issuestore/index.pb

```diff
@@ -120,4 +120,67 @@ gonboarding_auth/onboarding_auth_data/src/test/java/com/alexafit/onboarding_auth
 �
```

Comment:
You are uploading a .idea folder to git

You should have this added to your git ignore, since the .idea shouldn't be uploaded